augly/tests/image_tests/functional_unit_test.py

```diff
@@ -17,7 +17,10 @@ def test_blur(self):
 
     def test_brightness(self):
         self.evaluate_function(imaugs.brightness)
-
+        
+    def test_skew(self):
+        self.evaluate_function(imaugs.skew)
+        
```

Comment:
please insert this function in alphabetical order with respect to the other functions in this file

Reply:
Done!